homeassistant/components/vicare/strings.json

```diff
@@ -80,6 +80,15 @@
       },
       "comfort_temperature": {
         "name": "Comfort temperature"
+      },
+      "normal_heating_temperature": {
+        "name": "Normal temperature"
+      },
+      "reduced_heating_temperature": {
+        "name": "Reduced temperature"
+      },
+      "comfort_heating_temperature": {
+        "name": "Comfort temperature"
```

Comment:
You're duplicating the names here so how is the user going to know what is what?
Maybe add "heating" in the middle to diff from the ones above (or something else, maybe they have a good name in the app)?

Reply:
Devices support either one of them. Gas heatings have a *comfort* program where this in typically named *comfort_heating* on heat pumps as they have also a *comfort_cooling* program.

I thought it makes no sense to have the *heating* in the name as of today as we have no support for the `cooling` programs.

Reply:
I think we should mimic the native app/control so if it's "Comfort heating" I think we should say so.
Also later we don't want to change this if we implement the `cooling`s programs.

Lastly if none of above use a link to the previous translation to not duplicate for the translators (example: `"[%key:component::vicare::something::comfort_temperature::name%]"`)

Reply:
Just checked my system again, there is just the ability to use "comfort" and nothing about "heating"/"cooling" at all.
So I will add a link tho the existing translation.